src/modules/projection/dto/createProjectionDto.ts

```diff
@@ -0,0 +1,34 @@
+import { ProjectionType } from './projectionType';
+import { IsNotEmpty, IsNumber, IsString } from 'class-validator';
+import { Transform } from 'class-transformer';
+
+export class CreateProjectionDto {
+  @IsString()
+  @IsNotEmpty()
+  startTime: string;
+
+  @IsString()
+  @IsNotEmpty()
+  startDate: string;
```

Comment:
There is no need to store date and time as separate fields I believe, but you can keep it like that if it's more convienient for you, it does not affect anything